tests/jballerina-unit-test/src/test/resources/test-src/parser/resilient-parser.bal

```diff
@@ -0,0 +1,45 @@
+public function (int a, int b) {
+    int x1 = 2;
+}
+
+public function foo1(int) {
+    int x1 = 3;
+}
+
+public function foo2(int a, = "hello") {
+    int x1 = 3;
+}
+
+public function foo2(int a =) {
+    int x1 = 3;
+    int x1 = 3;
+}
+
+public function foo3(int a) {
+    return a;
+}
+
+public function foo4(int a) returns {
+}
+
+public function foo5(int a, = "hello", int b =) {
+    int x1 = 3;
+    int[] arr = [10, 20,];
+    arr[1 = 20;
+    int x = arr[1;
+    ar[2] = 40;
+    int y = arr1];
+}
+
+type Foo object {
+    int x;
+
+    function bar () returns int {
+        return 2;
+    }
+};
+
+public function foo6() {
+    Foo f  new;
+    int x = ff.bar();
```

Comment:
should this be `ff.bar()` or `f.bar()`?

Reply:
What I expect here is to get the error 'redeclared symbol'.